keras/engine/topology.py

```diff
@@ -2932,7 +2932,7 @@ def load_weights_from_hdf5_group(f, layers):
     for k, name in enumerate(layer_names):
         g = f[name]
         weight_names = [n.decode('utf8') for n in g.attrs['weight_names']]
-        weight_values = [g[weight_name] for weight_name in weight_names]
+        weight_values = [g[weight_name][...] for weight_name in weight_names]
```

Comment:
This seems unnecessary, and the ellipsis object being a niche indexing feature that few understand, it is not good to have as part of any codebase.